pkg/investigations/chgm/chgm.go

```diff
@@ -21,88 +21,77 @@ import (
 	v1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
 )
 
-// NOTE: USE CAUTION WHEN CHANGING THESE TEMPLATES!!
-// Changing the templates' summaries will likely prevent CAD from removing clusters with these Limited Support reasons in the future, since it identifies which reasons to delete via their summaries.
-// If the summaries *must* be modified, it's imperative that existing clusters w/ these LS reasons have the new summary applied to them (currently, the only way to do this is to delete the current
-// reason & apply the new one). Failure to do so will result in orphan clusters that are not managed by CAD.
-var chgmLimitedSupport = ocm.LimitedSupportReason{
-	Summary: "Cluster not checking in",
-	Details: "Your cluster is no longer checking in with Red Hat OpenShift Cluster Manager. Possible causes include stopped instances or a networking misconfiguration. If you have stopped the cluster instances, please start them again - stopping instances is not supported. If you intended to terminate this cluster then please delete the cluster in the Red Hat console",
+var chgmSL = ocm.ServiceLog{
+	Severity:     "Error",
+	Summary:      "Action required: cluster not checking in",
+	ServiceName:  "SREManualAction",
+	Description:  "Your cluster is no longer checking in with Red Hat OpenShift Cluster Manager. Possible causes include stopped instances or a networking misconfiguration. If you have stopped the cluster instances, please start them again - stopping instances is not supported. If you intended to terminate this cluster then please delete the cluster in the Red Hat console",
+	InternalOnly: false,
 }
 
-// CAUTION
-
 // InvestigateTriggered runs the investigation for a triggered chgm pagerduty event
 func InvestigateTriggered(r *investigation.Resources) error {
+	var notesSb strings.Builder
```

Comment:
In this function I did the following:
- inverted the logic to return early when user is not authorized
- used stringbuilder to build a nice note for primary and have the note in line with the CPD one (which is way less complex to understand)

Reply:
praise